bin/node/runtime/src/lib.rs

```diff
@@ -363,6 +364,8 @@ impl pallet_staking::Trait for Runtime {
 	type ElectionLookahead = ElectionLookahead;
 	type Call = Call;
 	type SubmitTransaction = TransactionSubmitterOf<()>;
+	type Randomness = RandomnessCollectiveFlip;
```

Comment:
Why not `Babe`?

Reply:
Ow I didn't know Babe also implements it. What are the differences? I don't think the randomness that Staking needs is in any sense consensus critical.

Reply:
`Babe` provides "real" randomness, while the one provided by collective flip is based on the last 80? block hashes.

Reply:
isn't there a local (non-deterministic) `rand()` function for offchain workers?

Reply:
Yeah.

Reply:
I'll switch to that then 👍